src/main/drivers/mcu/stm32/pwm_output.c

```diff
@@ -35,6 +35,10 @@
 
 #include "pg/motor.h"
 
+#ifndef PWM_SHUTDOWN_DELAY
+#define PWM_SHUTDOWN_DELAY 600
```

Comment:
This interval may be too short. You need at least 2ms for standard PWM pulse ( just for pulse to end, PWM interval may be 20ms)

Reply:
sorry, i confused us and ms.
.6 s is noticeable time, but esc disarm is reasonable strategy. Maybe delay can be moved outside of this function (there is already delay after shutdown call). I'll look into it

Reply:
I think it's ok to move delay to motorShutdown, but's not all protocols require such a long delay.
The delay value can be set based on the ESC protocol currently in use.
We are on holiday now, I'll change and test it later.

Reply:
You can return minimum guard time before reboot from `motorDevice->vTable.shutdown();`. Then extend delay in motorShutdown(); if necessary.